sdk/master_board_sdk/CMakeLists.txt

```diff
@@ -110,12 +102,14 @@ if(BUILD_PYTHON_INTERFACE)
   # --- Build the wrapper library --- #
   set(${PYWRAP}_INSTALL_DIR ${PYTHON_SITELIB})
   add_library(${PYWRAP} SHARED ${${PYWRAP}_SOURCES} ${${PYWRAP}_HEADERS})
-  target_include_directories(${PYWRAP} SYSTEM PRIVATE ${PYTHON_INCLUDE_DIR})
-  target_link_libraries(${PYWRAP} ${PROJECT_NAME})
-  target_link_boost_python(${PYWRAP})
-  get_relative_rpath(${${PYWRAP}_INSTALL_DIR} ${PYWRAP}_INSTALL_RPATH)
-  set_target_properties(${PYWRAP} PROPERTIES INSTALL_RPATH
-                                             "${${PYWRAP}_INSTALL_RPATH}")
+  set_target_properties(${PYWRAP} PROPERTIES SUFFIX ${PYTHON_EXT_SUFFIX})
+  target_link_libraries(${PYWRAP} PUBLIC ${PROJECT_NAME} eigenpy::eigenpy)
+  if(UNIX AND NOT APPLE)
```

Comment:
This should also work on apple, ref. https://github.com/stack-of-tasks/eigenpy/blob/master/python/CMakeLists.txt#L51

Reply:
Oh alright ! I was looking at aligator/proxsuite-nlp as reference. I'll update them and here.